content/english/events/20210501.md

```diff
@@ -16,6 +16,7 @@ keywords:
 tags: ["Fellowship"]
 math: false
 toc: true
+thumbnail: "/images/events/20210501.jpeg"
```

Comment:
on this file, please change the description to:

```
Rev. Dr. Stephen Tong - Saturday, May 1st, 2021
```

supaya sesuai juga sama yang indo.